### ISSUE TITLE
Wifi

### ISSUE THREAD

**Issue by adiyenuubarii** [Whitehathuzefa/FacebookPhishingPage#1]
Please make a wifi hacking software as well.

**Comment by Whitehathuzefa** [comment]
Yeah Sure, That's coming soon..

**Comment by adiyenuubarii** [comment]
And also how to set up a server for the snapchat phishing page?
Thanks and Regards,
Supriya Rao Yenubari

**Comment by Whitehathuzefa** [comment]
Yeah you need to upload it to a free server called webhost or you can upload it on your own apache web server

**Comment by Whitehathuzefa** [comment]
> Please make a wifi hacking software as well.

In which language you want?

**Comment by adiyenuubarii** [comment]
Any language which is easy to use

**Comment by adiyenuubarii** [comment]
Also can you tell me a site through which I can host?

**Comment by adiyenuubarii** [comment]
Can you tell me through which site I can host for Snapchat?

**Comment by Whitehathuzefa** [comment]
Its called 000webhostapp, you can check tutorial here :
https://youtu.be/TPngmEUL3Gk
